core/ui/legacy/theme2/README.md

```diff
@@ -0,0 +1,12 @@
+## Core - UI - Legacy - Theme 2
+
+Theme 2 represents a Material 3 adaptation of the legacy application theme. It follows the design principles of Material 3, while ensuring compatibility with the existing implementation.
```

Comment:
To avoid confusion between the existing "Theme 2" and this one, we could call this "Legacy Theme 2".

```suggestion
Legacy Theme 2 represents a Material 3 adaptation of the legacy application theme. It follows the design principles of Material 3, while ensuring compatibility with the existing implementation.
```